src/WebApiDebugger/WebApiDebugger.csproj

```diff
@@ -11,8 +11,8 @@
   <ItemGroup>
     <PackageReference Include="Swashbuckle.AspNetCore" Version="6.6.2" />
   </ItemGroup>
-
   <PropertyGroup>
+    <OpenApiEnabled>false</OpenApiEnabled>
```

Comment:
Prevent side effect when trying the debug profile. Otherwise before being able to debug it will first build the solution with the OpenAPI activated which will cause side-effect: like re-generating the spec, not running spectral since it has not changed since last time...